packages/shared/src/modules/groups/types/group-item-dto.type.ts

```diff
@@ -0,0 +1,10 @@
+import { type PermissionItemDto } from "../../permission/types/types.js";
+
+type GroupItemDto = {
```

Comment:
+ do not forget about file name

```suggestion
type GroupItemWithPermissionsDto = GroupResponseDto & { permissions: PermissionItemDto[]; }
```